src/CosmosDB/CosmosDB/CosmosDBAccount/NewOrUpdateAzCosmosDBAccount.cs

```diff
@@ -117,6 +117,9 @@ public class NewOrUpdateAzCosmosDBAccount : AzureCosmosDBCmdletBase
         [PSArgumentCompleter(SDKModel.MinimalTlsVersion.Tls, SDKModel.MinimalTlsVersion.Tls11, SDKModel.MinimalTlsVersion.Tls12)]
         public string MinimalTlsVersion { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = Constants.EnablePerRegionPerPartitionAutoscaleHelpMessage)]
+        public bool? EnablePerRegionPerPartitionAutoscale { get; set; }
```

Comment:
Refer to this [guide](https://github.com/Azure/azure-powershell/blob/674123b1e883dc764cfef1cb728e54f54ef9eab3/documentation/development-docs/design-guidelines/parameter-best-practices.md#bool-vs-switchparameter). Please use "SwitchParameter" instead.